packages/automation/README.md

```diff
@@ -0,0 +1,347 @@
+# @lit-protocol/automation
+
+A TypeScript library for creating and managing state machines that can automate complex workflows involving the Lit Protocol network, blockchain events, and other triggers.
```

Comment:
Perhaps we simplify this sentence (and #overview section) to:

> A TypeScript library for creating and managing automated workflows using Lit Protocol.
> 
> The automation package provides a flexible state machine implementation that allows you to:
> - Execute automated Lit Action workflows based on custom triggers, such as events on blockchains or off-chain platforms
> - Automate the minting of PKPs, Capacity Credits, and other Lit operations
> - Monitor token balances and price info
> - Perform cross-chain messaging and transaction execution
> - And more...

Reply:
Changed